doc/NEWS.md

```diff
@@ -247,7 +247,7 @@ Thanks to Manuel Mausz for to testing and improving portability!
 - make dbus example work again
 - fix compiler warnings for gcc and clang
 - fix valgrind suppressions
-- Installation of GI binding is fixed, thanks to Dāvis
+- Installation of GI binding is fixed, thanks to Davis
```

Comment:
Other parts of this PR is ok, but I think thats a name written like that. This should not be changed to ASCII.

Why exactly is this change needed?